### PR TITLE
Create OpenStack identity services module

### PR DESCRIPTION
**Pull request by ghost**
##### Issue Type:
- New Module Pull Request
##### Plugin Name:

os_keystone_service
##### Ansible Version:

`ansible 2.1.0`
##### Summary:

New module using shade to create and manage OpenStack identity
services

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @SamYaple for this new module. When this module receives 'shipit' comments from two community members and any 'needs_revision' comments have been resolved, we will mark for inclusion.

**Comment by gregdek** [comment]
Thanks @SamYaple for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by ghost** [comment]
I will be updating this with the final pieces needed from shade in the next week^H^H^H^H month

**Comment by gregdek** [comment]
@SamYaple A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@SamYaple Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by ghost** [comment]
ready_for_review

**Comment by gregdek** [comment]
Thanks @SamYaple. To the current maintainers, @emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by ghost** [comment]
It appears the error is unrelated to my patch

**Comment by andsens** [comment]
Fixed with #2380 :-)

**Comment by ghost** [comment]
Thanks @andsens !

**Comment by gregdek** [comment]
Thanks @SamYaple for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by ghost** [comment]
ready_for_review

@andsens Your patch worked for me. Tests are passing now that you fixed sensu. Thanks!

**Comment by gregdek** [comment]
Thanks @SamYaple. To the current maintainers, @emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/openstack/os_keystone_service.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+     description:
+        - Description of the service
+     required: false
+     default: None
+   enabled:
+     description:
+        - Is the service enabled
+     required: false
+     default: True
+   type:
```
Comment (Shrews) [inline cloud/openstack/os_keystone_service.py]: Can we call this service_type instead? Having it a python keyword is awkward.

Comment (ghost) [inline cloud/openstack/os_keystone_service.py]: We can, however it is named type in the keystone api (service_type is keystone v2) and this is the service module. I personally have no preference on this matter.

**Comment by Shrews** [comment]
shipit

**Comment by gregdek** [comment]
Thanks again to @SamYaple for this PR, and thanks @emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz for reviewing. Marking for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by rcarrillocruz** [comment]
shipit

**Pull request merged**